notifiers/pushoverNotifier.go

```diff
@@ -0,0 +1,159 @@
+package notifiers
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/multiversx/mx-chain-keys-monitor-go/core"
+
+	logger "github.com/multiversx/mx-chain-logger-go"
+	httpSDK "github.com/multiversx/mx-sdk-go/core/http"
+)
+
+const maxSendTimeout = time.Second * 30
+
+var log = logger.GetOrCreate("notifiers")
+
+type pushoverRequest struct {
+	Token   string `json:"token"`
+	User    string `json:"user"`
+	Title   string `json:"title"`
+	Message string `json:"message"`
+	HTML    int    `json:"html"`
+}
+
+type pushoverResponse struct {
+	Status  int    `json:"status"`
+	Request string `json:"request"`
+}
+
+type pushoverNotifier struct {
+	token             string
+	userKey           string
+	httpClientWrapper HTTPClientWrapper
+}
+
+// NewPushoverNotifier will create a new Pushover notifier
+func NewPushoverNotifier(url string, token string, userKey string) *pushoverNotifier {
+	return &pushoverNotifier{
+		httpClientWrapper: httpSDK.NewHttpClientWrapper(nil, url),
+		token:             token,
+		userKey:           userKey,
+	}
+}
+
+// OutputMessages will push the provided messages as error
```

Comment:
Not necessarily `as error`, right?

Reply:
right, refactored. Remained from previous tests/tries